### ISSUE TITLE
Bug while probing

### ISSUE THREAD

**Issue by sahmad98** [Morgan-Stanley/Xpedite#91]
When the binary is compiled with `-DXPEDITE_DISABLE` flag and `xpedite probe -p profile_info.py` is run following error is generated

```bash
➜  profiling xpedite probes -p profiling_info.py
Traceback (most recent call last):
  File "/home/saleem/source/Xpedite/scripts/bin/xpedite", line 251, in <module>
    app.run()
  File "/home/saleem/source/Xpedite/install/runtime/local/lib/python2.7/site-packages/cement/core/foundation.py", line 797, in run
    return_val = self.controller._dispatch()
  File "/home/saleem/source/Xpedite/install/runtime/local/lib/python2.7/site-packages/cement/ext/ext_argparse.py", line 901, in _dispatch
    return func()
  File "/home/saleem/source/Xpedite/scripts/bin/xpedite", line 111, in probes
    LOGGER.error('failed to locate probes in app %s. Have you instrumented any ?', profileInfo.appName)
NameError: global name 'profileInfo' is not defined

```

**Comment by sahmad98** [comment]
Raised a pull request for this bug #92

**Comment by sahmad98** [comment]
Fixed in #92
